File: election-api-javascript/src/resultsController.js
const { default: resultStore } = require('./resultsService');

const store = resultStore();

function getResult(id) {
    return store.getResult(id);
}

function newResult(result) {
    store.newResult(result);
    return {};
}

function reset() {
    store.reset();
}

function scoreboard() {
    //  Left blank for you to fill in
}


module.exports = { getResult, newResult, reset, scoreboard };

File: election-api-python/src/results_controller.py
from results_service import ResultStore

class ResultsController:

    def __init__(self) -> None:
        self.result_store: ResultStore = ResultStore()

    def get_result(self, identifier: int) -> dict:
        return self.result_store.get_result(identifier)

    def new_result(self, result: dict) -> dict:
        self.result_store.new_result(result)
        return {}

    def reset(self) -> None:
        self.result_store.reset()

    def scoreboard(self) -> dict:
        # Left blank for you to fill in
        return {}
